rest/blip_sync.go

```diff
@@ -570,7 +600,9 @@ func (bh *blipHandler) sendRevision(sender *blip.Sender, seq db.SequenceID, docI
 		outrq.Properties["deleted"] = "1"
 		delete(body, "deleted")
 	}
-	outrq.Properties["sequence"] = string(seqJSON)
+	if marshalErr == nil {
+		outrq.Properties["sequence"] = string(seqJSON)
+	}
```

Comment:
Looking at the spec, sequence is generally optional in a rev message, other than the case where a changes message hasn't been sent.  I don't think we have that scenario for rev messages sent by Sync Gateway.   If we can omit sequence altogether, we should file a ticket for that change for Iridium.

Reply:
Motivation is to reduce CPU cost on SG (marshalling sequence) and transport cost to clients.  Agree that this doesn't need to be changed for this PR.

If there are actually low-level troubleshooting benefits, we should address those via logging.  (although I don't know of any cases where id/rev could be associated with different sequences).

Reply:
Looking closer, the sequence that we're sending in the rev message is always just the sequence we previously sent in the changes message - we're not attempting to recalculate it from the document's current metadata.  Given that, I think there's even less value in sending it back to the client on the rev message.  I'm not sure why the changes response from CBL is specifying sequence, in fact.  Should be reviewed w/ https://github.com/couchbase/sync_gateway/issues/3764